cmd/terraform-demux/main.go

```diff
@@ -17,9 +18,15 @@ func main() {
 		log.SetOutput(ioutil.Discard)
 	}
 
-	log.Printf("terraform-demux version %s", version)
+	arch := os.Getenv("TF_DEMUX_ARCH")
 
-	exitCode, err := wrapper.RunTerraform(os.Args[1:])
+	if arch == "" {
+		arch = runtime.GOARCH
```

Comment:
should we do a validation on `arch`?

Reply:
I thought about that! But at the end of the day if we can't find a Terraform build for your `arch`, it will error out, and that feels good enough to me.